scripts/common_startup.sh

```diff
@@ -137,8 +139,28 @@ fi
 : ${GALAXY_WHEELS_INDEX_URL:="https://wheels.galaxyproject.org/simple"}
 if [ $REPLACE_PIP -eq 1 ]; then
     pip_version=`pip --version | awk '{print $2}'`
-    pre=`python -c "from pkg_resources import parse_version; from sys import stdout; stdout.write('--pre') if parse_version('$pip_version') >= parse_version('1.4') else stdout.write('')"`
-    pip install $pre --no-index --find-links ${GALAXY_WHEELS_INDEX_URL}/pip --upgrade pip
+    method=`python - << EOF
+from pkg_resources import parse_version
+from sys import stdout
+if parse_version('$pip_version') >= parse_version('1.6'):
```

Comment:
Should indeed be - it all worked as it was since there is no version >= 1.6 and <= 6.0